enterprise/functions/src/test/java/io/crate/window/NthValueFunctionIntegrationTest.java

```diff
@@ -46,13 +46,14 @@ public void testGeneralPurposeWindowFunctionsWithStandaloneValues() {
         execute("select col1, col2, " +
                 "first_value(col1) OVER(order by col2, col1), " +
                 "last_value(col1) OVER(order by col2, col1), " +
-                "nth_value(col1, 3) OVER(order by col2, col1) " +
+                "nth_value(col1, 3) OVER(order by col2, col1), " +
+                "lag(col1) OVER(order by col2, col1) " +
```

Comment:
I think this should have its own integration test as this is nth_value specific stuff only

Reply:
sure! didn't pay attention to it. Will rename `NthValueFunctionIntegrationTest` to `WindowFunctionIntegrationTest` and split tests.

Reply:
We have a `WindowFunctionsIntegrationTest` that aims to cover the framing/windowing combinations. Maybe a dedicated `LagFunctionIntegrtionTest` would work for this case?

Reply:
Do we even need a integration test per function? Are there concerns that the unit-test framework is not good enough?